srst2/0.2.0/README.md

```diff
@@ -0,0 +1,29 @@
+# SRST2 container
+
+Main tool : [SRST2](https://github.com/katholt/srst2) 0.2.0
+
+Additional tools:
+
+- Biopython 1.76
+- [Bowtie2](https://github.com/BenLangmead/bowtie2) 2.2.6-2
+- python 2.7
+-[SAMtools](https://github.com/samtools/samtools) 0.1.18
```

Comment:
missing a space after the `-` on line 10. check out the rendered markdownfile here to see what I'm referring to https://github.com/DOH-HNH0303/docker-builds/blob/master/srst2/0.2.0/README.md

Reply:
Fixed!

```
Additional tools:

- Biopython 1.76
- [Bowtie2](https://github.com/BenLangmead/bowtie2) 2.2.6-2
- python 2.7
- [SAMtools](https://github.com/samtools/samtools) 0.1.18
- SciPy 0.16
```

Reply:
thanks!